app-config.yaml

```diff
@@ -13,6 +13,26 @@ backend:
   database:
     client: sqlite3
     connection: ':memory:'
+  ingestionProcessors:
```

Comment:
Since this is technically specific to the catalog, would it make sense to have this config section nested under a new root key called "catalog" instead of "backend"?

I know there have been some discussions around structure and conventions for configs, so it's possible others might have alternatives conventions in mind.

Reply:
Been playing around with in idea to make repo reading a generic thing as we need it for techdocs and scaffolder too (at least). For now very much +1 for this though, and prolly just call it `processors`.

Reply:
thanks @andrewthauer and @Rugvip . moved the configuration to the existing root `catalog` and renamed to `processors`. is there anything else you want me to change.